cli/args/flags.rs

```diff
@@ -4658,8 +4689,21 @@ fn check_parse(
   Ok(())
 }
 
-fn clean_parse(flags: &mut Flags, _matches: &mut ArgMatches) {
-  flags.subcommand = DenoSubcommand::Clean;
+fn clean_parse(flags: &mut Flags, matches: &mut ArgMatches) {
```

Comment:
In this file there are unit tests for flag parsing. Maybe we should add some for the clean command now?

Reply:
Agreed, especially that the parsing is a bit quirky